webpack.config.js

```diff
@@ -0,0 +1,51 @@
+import {UserscriptPlugin} from 'webpack-userscript';
+
+export default {
+  mode: 'production',
+  devtool: 'source-map',
+  entry: {
+    loader: './loader.ts',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        loader: 'ts-loader',
+        options: {
+          instance: 'loader',
+          configFile: "./tsconfig.json"
+        },
+      },
+    ],
+  },
+  output: {
+    filename: '[name].js',
+    library: {
+      type: 'module'
+    }
+  },
+  plugins: [new UserscriptPlugin({
+    headers: {
+      name: "r/place MLP Minimap",
+      author: "r/place MLP Minimap Authors",
+      description: "Minimap overlay and bot for r/place",
+      namespace: "http://tampermonkey.net/",
```

Comment:
```suggestion
      namespace: "https://tampermonkey.net/",
```
unless there's some reason not to